hh_aidoc[ESX]/client.lua

```diff
@@ -1,51 +1,51 @@
-local ESX = nil
-
-Citizen.CreateThread(function()
-	while ESX == nil do
-	   TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
-	   Citizen.Wait(200)
-	end
-end)
-
+ESX = nil
 local Active = false
 local test = nil
 local test1 = nil
 local Act = true
 local spam = true
-
 local isDead = false
 
+Citizen.CreateThread(function()
+    while ESX == nil do
+        TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
+        Citizen.Wait(200)
+    end
+end)
 
 AddEventHandler('playerSpawned', function(spawn)
 	isDead = false
 end)
 
 AddEventHandler('esx:onPlayerDeath', function(data)
 	isDead = true
-end)	
-
-
-
-RegisterCommand("help", function(source, args, raw)
-	if isDead and spam then
-		ESX.TriggerServerCallback('hhfw:docOnline', function(EMSOnline, hasEnoughMoney)
-
-			if EMSOnline <= Config.Doctor and hasEnoughMoney and spam then
-				SpawnVehicle(GetEntityCoords(PlayerPedId()))
-				TriggerServerEvent('hhfw:charge')
-				Notify("Medic is arriving")
-			else
-				if EMSOnline > Config.Doctor then
-					Notify("There is too many medics online", "error")
-				elseif not hasEnoughMoney then
-					Notify("Not Enough Money")
-				else
-					Notify("Wait Paramadic is on its Way")
-				end	
+end)
+
+
+Citizen.CreateThread(function()
+	while true do
+		Citizen.Wait(0)
+		if isDead and spam then
+			if IsControlJustPressed(1, 47) then
+				ESX.TriggerServerCallback('hhfw:docOnline', function(EMSOnline, hasEnoughMoney)
+					if EMSOnline <= Config.Doctor and hasEnoughMoney and spam then
+						SpawnVehicle(GetEntityCoords(PlayerPedId()))
+						TriggerServerEvent('hhfw:charge')
+						ShowAdvNotification("CHAR_PA_FEMALE", "Doctor", "", "There is no EMS in the city don't worry i will send my PA for you!")
+						--Notify("Medic is arriving")
+					else
+						if EMSOnline > Config.Doctor then
+							exports['CRP_Notify']:Alert("Doctor", "We got your signal, please wait", 5000, 'bell')
```

Comment:
Better to leave this with as a comment and uncomment the notify under this. This way there's no need to implement a dependency.